TTS/bin/compute_embeddings.py

```diff
@@ -22,11 +23,13 @@
 parser.add_argument("config_dataset_path", type=str, help="Path to dataset config file.")
 parser.add_argument("--output_path", type=str, help="Path for output `pth` or `json` file.", default="speakers.pth")
 parser.add_argument("--old_file", type=str, help="Previous embedding file to only compute new audios.", default=None)
-parser.add_argument("--use_cuda", type=bool, help="flag to set cuda. Default False", default=False)
+parser.add_argument("--disable_cuda", type=bool, help="Flag to disable cuda.", default=False)
```

Comment:
To me this is counter-intuitive but can live it with that :)

Reply:
I just spent 3h computing phonemes before realizing that it wasn't using the GPU ^^' That's why I changed it to that. Having use_cuda to true by default and then throwing errors if you had no GPU seemed bad...